README.md

```diff
@@ -163,6 +163,74 @@ python buildbot/compile.py -o build
+```
+
+Follow the instructions in the installer. And explore the Get Started Guide to get more information.
+
+For example, for Linux, online installer and version 2023.2.0, follow the instructions below:
+
+```sh
+wget https://registrationcenter-download.intel.com/akdlm/IRC_NAS/992857b9-624c-45de-9701-f6445d845359/l_BaseKit_p_2023.2.0.49397.sh
+
+sudo sh ./l_BaseKit_p_2023.2.0.49397.sh
+```
+
+Now to compile the vector add from oneAPI samples mentioned in previous section, using the official Intel base toolkit, set the environment variables and follow the stepsgiven below:
+
+```sh
+export OCL_ICD_FILENAMES=$ONEAPI_CON_KIT_INSTALL_DIR/lib/libCL.so
+export LD_LIBRARY_PATH=/path/to/intel/oneapi/compiler/2023.2.0/linux/lib:/path/to/intel/oneapi/compiler/2023.2.0/linux/compiler/lib/intel64_lin:/path/to/intel/oneapi/compiler/2023.2.0/linux/compiler/lib/:$LD_LIBRARY_PATH
+export ONEAPI_DEVICE_SELECTOR="*:acc"
+
+/path/to/intel/oneapi/compiler/2023.2.0/linux/bin-llvm/clang++ -fsycl /path/to/oneAPI-samples/DirectProgramming/C++SYCL/DenseLinearAlgebra/vector-add/src/vector-add-buffers.cpp -o vector-add-buffers
+CA_HAL_DEBUG=1 SYCL_CONFIG_FILE_NAME=
+```
+
+>**_Note_:**
+   As the release has a whitelist of devices, it filters out RefSi. To override it, as a temporary solution we can point `SYCL_CONFIG_FILE_NAME` to empty file. This way it doesn't set the default `sycl.conf`.
+
+The generated output should be somthing like the following:
+```sh
+Running on device: RefSi G1 RV64
+Vector size: 10000
+refsi_hal_device::mem_alloc(size=40000, align=128) -> 0x98006380
+refsi_hal_device::mem_write(dst=0x98006380, size=40000)
+refsi_hal_device::mem_alloc(size=40000, align=128) -> 0x97ffc700
+refsi_hal_device::mem_write(dst=0x97ffc700, size=40000)
+refsi_hal_device::mem_alloc(size=40000, align=128) -> 0x97ff2a80
+refsi_hal_device::mem_write(dst=0x97ff2a80, size=40000)
+refsi_hal_device::program_find_kernel(name='_ZTSZZ9VectorAddRN4sycl3_V15queueERKSt6vectorIiSaIiEES7_RS5_ENKUlRNS0_7handlerEE_clESA_EUlT_E_.mux-kernel-wrapper') -> 0x00010570
+refsi_hal_device::kernel_exec(kernel=0x00010570, num_args=6, global=<10000:1:1>, local=<16:1:1>)
+refsi_hal_device::pack_arg(offset=0, align=8, value=0x0000000097ff2a80)
+refsi_hal_device::pack_arg(offset=8, align=8, value=0x0000000000000000)
+refsi_hal_device::pack_arg(offset=16, align=8, value=0x0000000098006380)
+refsi_hal_device::pack_arg(offset=24, align=8, value=0x0000000000000000)
+refsi_hal_device::pack_arg(offset=32, align=8, value=0x0000000097ffc700)
+refsi_hal_device::pack_arg(offset=40, align=8, value=0x0000000000000000)
+refsi_hal_device::kernel_exec finished in 0.003 s
+refsi_hal_device::mem_read(src=0x97ff2a80, size=40000)
+refsi_hal_device::mem_free(address=0x97ff2a80)
+refsi_hal_device::mem_free(address=0x97ffc700)
+refsi_hal_device::mem_free(address=0x98006380)
+[0]: 0 + 0 = 0
+[1]: 1 + 1 = 2
+[2]: 2 + 2 = 4
+...
+[9999]: 9999 + 9999 = 19998
+Vector add successfully completed on device.
+```
+
```

Comment:
I wonder if it is also worth updating the part about the binary version of clang++ which is tied to quite an old version - maybe update to a recent one and say this updates daily.